ui/store/actions.ts

```diff
@@ -4137,15 +4136,15 @@ export function createEventFragment(
   ]);
 }
 
-export function createTransactionEventFragment(
+export function createTransactionEventFragmentWithTxId(
```

Comment:
As mentioned, should we keep the name the same for consistency sake and to avoid the 12 UI changes in this PR?

Reply:
Renamed all in https://github.com/MetaMask/metamask-extension/pull/21330/commits/22e50c9635789db378ed0c925a3e1eebefc28c8b